Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/view_routes.py

```diff
@@ -169,5 +171,6 @@ def constants(self):
         return Response(
             self.environment.get_template('constants.js').render(
                 XcodeCloud=self.suite_types.get('XcodeCloud') or [],
+                default_architecture=f"'{self.default_architecture}'" if self.default_architecture else 'null',
```

Comment:
```suggestion
                default_architecture=json.dumps(self.default_architecture),
```

…would make me slightly less nervous as it ensures we get a valid JS type out, properly escaped. (And yes, we shouldn't have anything malicious in here, but the cost of doing it properly seems low!)